### PR TITLE
Smart assist readme fixes

### PR DESCRIPTION
**Pull request by Har1pr1ya**
- Updated ReadMe with deployment steps (in case of new bot scenario)
- Uploaded Screenshots for more clarity

### CODE REVIEW

File: customer-service/omnichannel/smart-assist-bot/README.md [hunk 1]
```diff
@@ -1,14 +1,158 @@
 # Build a custom smart assist bot
 
+### Getting Started
+This sample code contains custom smart bot implementation for suggesting KB articles and for Natural Language Understanding using [LUIS](https://www.luis.ai/home).
```
Comment (girimam) [inline customer-service/omnichannel/smart-assist-bot/README.md]: This sample code contains custom smart bot implementation for suggesting KB articles and appointment using LUIS provided by Azure Cognitive services.

File: customer-service/omnichannel/smart-assist-bot/README.md [hunk 2]
```diff
@@ -1,14 +1,158 @@
...
+- Go to directory **Dynamics365-Apps-Samples\customer-service\omnichannel\smart-assist-bot\SmartAssistBot**
+- Open the solution file [SmartBot.sln](https://github.com/microsoft/Dynamics365-Apps-Samples/blob/master/customer-service/omnichannel/smart-assist-bot/SmartAssistBot/CoreBot.sln) in Microsoft Visual Studio 2017 or above
+
+This setup broadly contains the following steps.
+
+1) Update the values in appsettings.json (SmartAssistBot/appsettings.json) file<br />
+2) Publish the webapp to Azure<br />
+3) Update the Messaging endpoint in Bot channels registration<br />
+4) Enable Teams channel<br />
+5) Test your Bot online<br />
```
Comment (girimam) [inline customer-service/omnichannel/smart-assist-bot/README.md]: Add a step to link back to bot configuration for Smart assist.

File: customer-service/omnichannel/smart-assist-bot/README.md [hunk 3]
```diff
@@ -1,14 +1,158 @@
...
+    ```
+    // Bot settings
+    "MicrosoftAppId": "<Microsoft App Id as copied in step 2d>",
+    "MicrosoftAppPassword": "<Microsoft App password as copied in step 2c>",
+    "REMOTEDEBUGGINGVERSION": "15.0.28307.222",
+    "ScmType": "None"```
+
+3) **CDS settings**
+
+    a. Login to Azure portal with CDS credentials<br />
```
Comment (girimam) [inline customer-service/omnichannel/smart-assist-bot/README.md]: Login withe tenant credentials?

File: customer-service/omnichannel/smart-assist-bot/README.md [hunk 4]
```diff
@@ -1,14 +1,158 @@
...
+![](screenshots/channel1.png)
+
+![](screenshots/Teams2.png)
+
+**Test your Smart bot**
+
+- Test your bot online using ```Test in Web Chat``` option. See [here](https://docs.microsoft.com/en-us/azure/bot-service/abs-quickstart?view=azure-bot-service-4.0#test-the-bot) for more details. 
+- If you are using trail org use keywords `travel`  or `book` or `audio equipment` to see the smart suggestions
+
+### 2 . Integrating Smart Assist to an exisiting Azure Bot
```
Comment (girimam) [inline customer-service/omnichannel/smart-assist-bot/README.md]: Consult whether it makes sense to have a section 2

File: customer-service/omnichannel/smart-assist-bot/README.md (hunk 3 above)
Comment (Har1pr1ya) [reply customer-service/omnichannel/smart-assist-bot/README.md]: Tested with non admin user and it worked fine.

File: customer-service/omnichannel/smart-assist-bot/README.md (hunk 2 above)
Comment (Har1pr1ya) [reply customer-service/omnichannel/smart-assist-bot/README.md]: Done !

File: customer-service/omnichannel/smart-assist-bot/README.md (hunk 4 above)
Comment (Har1pr1ya) [reply customer-service/omnichannel/smart-assist-bot/README.md]: Will check with Neerja and close this in next PR

File: customer-service/omnichannel/smart-assist-bot/README.md (hunk 1 above)
Comment (Har1pr1ya) [reply customer-service/omnichannel/smart-assist-bot/README.md]: Done!

**Pull request merged**